gratipay/models/participant/__init__.py

```diff
@@ -341,6 +341,14 @@ def clear_takes(self, cursor):
         """Leave all teams by zeroing all takes.
         """
         for team in self.get_teams():
+            if team.owner == self.username and team.get_take_for(self) == 0:
+
+                # Team owners don't necessarily have takes, and we also don't
+                # hard-require team owners to have a national identity on
+                # file. But we *do* require national identity before we will
+                # set a take--even if setting it to zero.
+
+                continue
```

Comment:
Hmm, this case only arises because we're pulling teams from `self.get_teams`, which considers both teams that a participant is an owner of and those where the participant is a member (code below). 

```python
teams = (cursor or self.db).all("""
            SELECT teams.*::teams FROM teams WHERE owner=%s

            UNION

            SELECT teams.*::teams FROM teams WHERE id IN (
                SELECT team_id FROM current_takes WHERE participant_id=%s
            )
        """, (self.username, self.id))
```

What we need... is to only pull the teams where the user has an active take. Might be worth splitting into a separate query?